etna/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       - ../docker/etna-base/entrypoints:/entrypoints:ro
       - /etc/passwd:/etc/passwd:ro
       - /etc/group:/etc/group:ro
+      - $HOME/etna.yml:/root/etna.yml
```

Comment:
Not, entirely convinced this is wise.  If you don't have etna.yml when you start this up, it creates a directory lol.

Reply:
Why would you want to mount etna.yml anyway?

Reply:
Eh, for testing the etna command stuff more conveniently.

Reply:
oh ah - yeah i have been just running gem build/install for testing, but it is painful.